src/libaktualizr/storage/sqlstorage.cc

```diff
@@ -1111,16 +1109,27 @@ class SQLTargetWHandle : public StorageTargetWHandle {
         sha512Hash = hash.HashString();
       }
     }
-    auto statement = db_.prepareStatement<std::string, SQLZeroBlob>(
-        "INSERT OR REPLACE INTO target_images (filename, image_data, sha256, sha512) VALUES (?, ?, ?, ?);",
-        target_.filename(), SQLZeroBlob{target_.length()}, sha256Hash, sha512Hash);
+    auto statement = db_.prepareStatement<std::string>("SELECT rowid FROM target_images_data WHERE filename = ?;",
```

Comment:
This statement seems to not be used later in the code. Should the line be deleted?

Reply:
Yes, have removed it